uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-balance-tab.component.spec.js

```diff
@@ -180,6 +194,23 @@ define([
         });
       });
 
+      describe('contacts', function () {
+        beforeEach(function () {
+          setupController();
+          $rootScope.$digest();
+        });
+
+        it('loads the contacts sorted by sort name', function () {
+          expect(Contact.all).toHaveBeenCalledWith({
+            options: { sort: 'sort_name ASC' }
+          });
+        });
+
+        it('stores the absence periods', function () {
```

Comment:
why is this test under contacts? I think you have not updated this test.

Reply:
Oops, fixed.